src/aleph/vm/orchestrator/views/__init__.py

```diff
@@ -199,6 +199,9 @@ async def status_check_fastapi(request: web.Request, vm_id: Optional[ItemHash] =
                 "index": await status.check_index(session, fastapi_vm_id),
                 "environ": await status.check_environ(session, fastapi_vm_id),
                 "messages": await status.check_messages(session, fastapi_vm_id),
+                # Using the remote account currently causes issues
+                # "post_a_message": await status.check_post_a_message(session, fastapi_vm_id),
+                # "sign_a_message": await status.check_sign_a_message(session, fastapi_vm_id),
```

Comment:
Why create that methods (`check_post_a_message` and `check_sign_a_message`) that aren't used finally on the fastapi endpoint?

Reply:
Because I am afraid we still have the issue with the `RemoteAccount` not working as we had when running with pytest. I don't want to make that a blocking issue.